tests/Resources/Page/ODH/Prometheus/Prometheus.robot

```diff
@@ -17,6 +17,27 @@ Run Query
     # Log To Console    ${resp.json()}
     [Return]    ${resp}
 
+Run Query_Range
+    [Documentation]    Runs a Prometheus query range using the API
```

Comment:
I think it could be renamed to `Run Range Query`

```
[Documentation]  Runs a prometheus range query, in order to obtain the result of a PromQL expression over a given time range. More info at:
... - https://promlabs.com/blog/2020/06/18/the-anatomy-of-a-promql-query
... - https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries
```